README.md

```diff
@@ -1,2 +1,76 @@
-# Hello-world
-a simple hello-world project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World 
+
+> A simple Hello World project.
+
+## Built With
+
+- Html,Css
+- Stylelint
+- Github actions
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+-node js & npm
+
+### Setup
+-git clone \<project-name>
+-cd \<project-name>
+
+### Install
+- nom Install
+
+<!--### Usage -->
+
+<!--### Run tests-->
+
+<!--### Deployment-->
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused line of code in order to have a clean code-base